ts/features/payments/common/components/PaymentCardSmall.tsx

```diff
@@ -9,13 +9,12 @@ import * as React from "react";
 import { Pressable, StyleSheet, View } from "react-native";
 import Animated from "react-native-reanimated";
 import Placeholder from "rn-placeholder";
-import { BrandEnum } from "../../../../../definitions/pagopa/ecommerce/WalletInfoDetails";
 import { useSpringPressScaleAnimation } from "../../../../components/ui/utils/hooks/useSpringPressScaleAnimation";
 import { WithTestID } from "../../../../types/WithTestID";
 import { LogoPaymentWithFallback } from "../../../../components/ui/utils/components/LogoPaymentWithFallback";
 
 type RenderData = {
-  iconName: IOLogoPaymentType | BrandEnum | undefined;
```

Comment:
why remove `BrandEnum` here?

Reply:
Required by an API updated, we do not have `BrandEnum` anymore